plots/parser.py

```diff
@@ -98,9 +104,18 @@ def prod(self, items):
          | subscriptsuperscript
          | superscriptsubscript
 
+GREEK_LETTER_NAME : "\\Alpha" | "\\Beta" | "\\Gamma" | "\\Delta" | "\\Epsilon" | "\\Zeta"
```

Comment:
Could you do the same for this too? I'm not sure what the best method would be, perhaps using an f-string

Reply:
I think i have solved it.